components/check/src/checkfile.py

```diff
@@ -0,0 +1,83 @@
+# -----
+#
+# Check the input file
+#
+# -----
+
+import pandas as pd
```

Comment:
where is `checkfile` called?

Reply:
So far, it's just a function. I wanted to leave it like this so that we can decide in which `main` it's called. It can be in this folder, or Jaczek might want to use it somewhere else. In addition, I don't know how the input file will be viewed by the app once uploaded. Since I didn't know about the pipeline, I left it as generic as possible, to be adjusted as necessary

Reply:
Yeah, it is just a function for now, but we need to call it somewhere. The check needs to execute, similar to the execution of the other pipeline functions.

Reply:
Yes I agree, but I wanted to coordinate with you first to understand how to call it ;) 
I can create a `main_check`, but what's exactly the input? Path to a specific input file from "input/test", or a wildcard that takes any input file (like `input/test/*.xlsx`)? It just depends on how we want to make tests afterwards

Reply:
For the tests later on, we can then change that file. Maybe define that as a variable so that we can easily change it, but by default, it is `input/input-data.xlsx`.